q2_diversity/tests/test_ordination.py

```diff
@@ -52,3 +53,37 @@ def test_pcoa_biplot(self):
 
         skbio.util.assert_ordination_results_equal(observed, expected,
                                                    ignore_directionality=True)
+
+
+class tsneTests(unittest.TestCase):
+
+    def setUp(self):
+        self.dm = skbio.DistanceMatrix([[0.0000000, 0.3333333, 0.6666667],
+                                        [0.3333333, 0.0000000, 0.4285714],
+                                        [0.6666667, 0.4285714, 0.0000000]],
+                                       ids=['S1', 'S2', 'S3'])
+
+    def test_tsne_default(self):
+        observed = tsne(self.dm).samples.shape
+        self.assertEqual(observed, (3, 3))
+
+    def test_tsne__custom(self):
+        observed_second = tsne(self.dm, 2).samples.shape
```

Comment:
Is this test testing the exact same path as the test above? I'm a little confused, because the default number of dimensions is `2` (as specified in your implementation function, above). Perhaps this test should be testing the number of dimensions value of `3`, instead?